Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -300,7 +304,11 @@ struct _WebKitWebViewBasePrivate {
     CString tooltipText;
     IntRect tooltipArea;
     WebHitTestResultData::IsScrollbar mouseIsOverScrollbar;
-#if !USE(GTK4)
+#if USE(GTK4)
+#if defined(GTK_ACCESSIBILITY_ATSPI)
```

Comment:
Style nit: when there is only one value to check, write `#ifdef foo` rather than `#if defined(foo)`:

```suggestion
#ifdef GTK_ACCESSIBILITY_ATSPI
```

Reply:
Done